unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -13,27 +11,33 @@
 /// </summary>
 public class ExploreV2MenuComponentController : IExploreV2MenuComponentController
 {
-    internal const float MIN_TIME_AFTER_CLOSE_OTHER_UI_TO_OPEN_START_MENU = 0.1f;
-
-    internal UserProfile ownUserProfile => UserProfile.GetOwnUserProfile();
-    internal RectTransform topMenuTooltipReference { get => view.currentTopMenuTooltipReference; }
-    internal RectTransform placesAndEventsTooltipReference { get => view.currentPlacesAndEventsTooltipReference; }
-    internal RectTransform backpackTooltipReference { get => view.currentBackpackTooltipReference; }
-    internal RectTransform mapTooltipReference { get => view.currentMapTooltipReference; }
-    internal RectTransform builderTooltipReference { get => view.currentBuilderTooltipReference; }
-    internal RectTransform questTooltipReference { get => view.currentQuestTooltipReference; }
-    internal RectTransform settingsTooltipReference { get => view.currentSettingsTooltipReference; }
-    internal RectTransform profileCardTooltipReference { get => view.currentProfileCardTooltipReference; }
+    internal const ExploreSection DEFAULT_SECTION = ExploreSection.Explore;
 
-    internal IExploreV2MenuComponentView view;
-    internal IPlacesAndEventsSectionComponentController placesAndEventsSectionController;
-    internal IExploreV2Analytics exploreV2Analytics;
-    internal ExploreSection currentOpenSection;
+    internal const float MIN_TIME_AFTER_CLOSE_OTHER_UI_TO_OPEN_START_MENU = 0.1f;
+    internal readonly List<RealmRowComponentModel> currentAvailableRealms = new List<RealmRowComponentModel>();
+    internal float chatInputHUDCloseTime = 0f;
     internal float controlsHUDCloseTime = 0f;
+    internal ExploreSection currentOpenSection;
     internal float emotesHUDCloseTime = 0f;
+    internal IExploreV2Analytics exploreV2Analytics;
+    internal MouseCatcher mouseCatcher;
+    internal IPlacesAndEventsSectionComponentController placesAndEventsSectionController;
     internal float playerInfoCardHUDCloseTime = 0f;
-    internal float chatInputHUDCloseTime = 0f;
-    internal List<RealmRowComponentModel> currentAvailableRealms = new List<RealmRowComponentModel>();
+
+    private Dictionary<BaseVariable<bool>, ExploreSection> SectionsByInitVar;
+    private Dictionary<BaseVariable<bool>, ExploreSection> SectionsByVisiblityVar;
```

Comment:
Lets follow the same variables naming style as the rest of the code:
```suggestion
    private Dictionary<BaseVariable<bool>, ExploreSection> sectionsByInitVar;
    private Dictionary<BaseVariable<bool>, ExploreSection> sectionsByVisiblityVar;
```

Reply:
good catch on private fields naming notation. I don't think it was my IDE mistake, but more probably I messed up with the naming 😅